aviary/utils/engine_deck_conversion.py

```diff
@@ -256,26 +256,26 @@ def EngineDeckConverter(input_file, output_file, data_format: EngineDeckType):
             # TODO flight condition dependent throttle range?
             # NOTE this often leaves max throttles less than 1 in the deck - this caues
             #     problems when finding reference SLS thrust, as there is often no max
-            #     power data at that point in the engine deck. It is reccomended GASP
+            #     power data at that point in the engine deck. It is recommended GASP
             #     engine decks override Aircraft.Engine.REFERENCE_THRUST in EngineDecks
             data[THROTTLE] = normalize(data[TEMPERATURE], minimum=0.0, maximum=t4max)
 
-            # remove all points above T4max
-            # TODO save these points as commented out?
-            valid_idx = np.where(data[THROTTLE] <= 1.0)
-            data[MACH] = data[MACH][valid_idx]
-            data[ALTITUDE] = data[ALTITUDE][valid_idx]
-            data[THROTTLE] = data[THROTTLE][valid_idx]
-            data[FUEL_FLOW] = data[FUEL_FLOW][valid_idx]
+        else:
+            data[THROTTLE] = normalize(T4T2, minimum=0.0, maximum=t4max)
```

Comment:
is t4max the right value to use as the maximum here? Is it actually the max t4/t2 if we are in this branch of the the `if`?

Reply:
What's happening here is if we are not calculating T4, that means that the T4T2 values are actually throttles. And based on the GASP engine decks we have in Aviary, in that scenario the t4max values correspond to max throttle (like in `turbofan_24k_2.deck`, the recovered t4max is 50.0 which perfectly lines up with max throttle)

I guess the normalization here is actually optional (Aviary will do it anyway after loading the data), so you can make the call if we want converted engines to maintain the original throttle values for easier comparison with GASP decks, or convert for 0-1 for easier comparisons with Aviary mission results